blockedModeSetup.js

```diff
@@ -2,6 +2,12 @@ const ENTER_KEY_CODE = 13;
 
 document.addEventListener('DOMContentLoaded', function() {
 	let textField = document.getElementById('duration');
+    let startButton = document.getElementById('start');
```

Comment:
On more tab here is needed?

Reply:
To github coś popsuł znów chyba, "u mnie działa". Może z kodowaniem dokumentu w n++ musimy poeksperymentować